### DISCUSSION TITLE
when support hot reload?

### DISCUSSION THREAD

**Discussion by niltor** [CommunityToolkit/Maui.Markup#188] [Q&A]
## Environment 
- System: Win 11
- IDE:VS 17.4.5
- SDK: .NET 7.03
- Project Type: MAUI

## Package Version 
```xml
  <PackageReference Include="CommunityToolkit.Maui" Version="3.1.0" />
  <PackageReference Include="CommunityToolkit.Maui.Markup" Version="2.1.0" />
  <PackageReference Include="CommunityToolkit.MVVM" Version="8.1.0" />
```

## Question
I use C# markup, but hot reload is not working. I searched for this problem but did not find useful things.

So, does hot reload not work with C# markup? If it's true, does anyone know when it will be supported?

If it is already supported, how to enable it?

**Comment by HobDev** [comment]
It's unfortunate that the hot reload does not work with C# UI. I reported the bug [here](https://developercommunity.visualstudio.com/t/The-C-UI-does-not-HotReload/10079380). You can upvote and track the issue from there.

**Comment by TheCodeTraveler** [comment]
Closing this to consolidate the Hot Reload discussions here: https://github.com/CommunityToolkit/Maui.Markup/discussions/71
